app/helpers/client_helper.rb

```diff
@@ -1,33 +1,27 @@
 # Contains functions which make client selection/branching easier
 module ClientHelper
-  def client_partial(client_name, path, args={})
-    client_name ||= "default"
-    to_check = client_name + "/" + path
+  def client_partial(client_slug, path, args={})
+    client_slug ||= "default"
+    to_check = client_slug + "/" + path
     default_check = "default/" + path
+
     if lookup_context.template_exists?(to_check, [], true)
       args[:partial] = to_check
       render(args)
     elsif lookup_context.template_exists?(default_check, [], true)
       args[:partial] = default_check
       render(args)
-    else
-      ""
```

Comment:
It does. At the rate we are adding client models, I suppose I'm not _too_ concerned about missing a partial somewhere. Fine for now.